app/services/notification_service.rb

```diff
@@ -7,32 +7,163 @@ def self.renderer
     )
   end
 
+  def self.get_template_for_event_type(event_type)
+    'map_mailer/' + event_type
+  end
+  
+  def self.get_mailboxer_code_for_event_type(event_type)
+    case event_type
+      when 'access_approved'
+        MAILBOXER_CODE_ACCESS_APPROVED
+      when 'access_request'
+        MAILBOXER_CODE_ACCESS_REQUEST
+      when 'invite_to_edit'
+        MAILBOXER_CODE_INVITE_TO_EDIT
+
+      when 'map_activity'
+        MAILBOXER_CODE_MAP_ACTIVITY
+      when 'map_collaborator_added'
+        MAILBOXER_CODE_MAP_COLLABORATOR_ADDED
+      when 'map_updated'
+        MAILBOXER_CODE_MAP_UPDATED
+      when 'map_message'
+        MAILBOXER_CODE_MAP_MESSAGE
+      when 'map_starred'
+        MAILBOXER_CODE_MAP_STARRED
+        
+      when 'topic_added_to_map'
+        MAILBOXER_CODE_TOPIC_ADDED_TO_MAP
+      when 'topic_connected'
+        MAILBOXER_CODE_TOPIC_CONNECTED
+      when 'topic_deleted'
+        MAILBOXER_CODE_TOPIC_DELETED
+      when 'topic_disconnected'
+        MAILBOXER_CODE_TOPIC_DISCONNECTED
+      when 'topic_updated'
+        MAILBOXER_CODE_TOPIC_UPDATED
+    end
+  end
```

Comment:
I'm not opposed to using these event names in mailboxer. but i also see the argument for leaving it as is, in case (or when) the events and the notifications begin to diverge more and more